linux_os/guide/services/http/securing_httpd/httpd_configure_log_format/rule.yml

```diff
@@ -22,8 +22,6 @@ severity: medium
 identifiers:
     cce@rhel7: CCE-80548-1
 
-references:
-    stigid: WA00612
```

Comment:
Do we want to remove these?

Reply:
I think yes, because they don't correspond to any of the STIG profiles that we have in our project.

Reply:
I proposed a similar thing in https://github.com/ComplianceAsCode/content/pull/11593/commits/1ad84d67c846406dc0b9ad230345c2032b5396f1